parser/sql/dialect/mysql/src/main/antlr4/imports/mysql/DMLStatement.g4

```diff
@@ -231,7 +231,7 @@ tableStatement
     ;
 
 tableValueConstructor
-    : VALUES rowConstructorList
+    : VALUES rowConstructorList orderByClause? limitClause?
```

Comment:
MySQL may not implement the "VALUES" statement in this way.
https://github.com/mysql/mysql-server/blob/1bfe02bdad6604d54913c62614bde57a055c8332/sql/sql_yacc.yy#L10051-L10056